x-pack/plugins/security_solution/server/lib/sourcerer/routes/index.ts

```diff
@@ -26,85 +35,220 @@ export const createSourcererDataViewRoute = (
         body: buildRouteValidation(sourcererSchema),
       },
       options: {
+        authRequired: true,
```

Comment:
Why was this added? I think the default value is `true` here according to the docs.

Reply:
I will remove it